src/components/Consents/Consents.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Consent from 'components/Consents';
+import 'i18NextInit';
+
+const store = (
+  loading = false,
+  error = '',
+  definitions = [],
+  labels = [],
+  checked = []
+) => ({
+  consents: {
+    definitions,
+    labels,
+    checked,
+    loading,
+    error
+  },
+  publisherConfig: { publisherId: '' }
+});
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+
+describe('PaymentInfo component', () => {
+  test('should render GeneralErrorStyled (if loading = false)', async () => {
```

Comment:
I I would specify here that 'should render GeneralErrorStyled when publisherId is not given)

Reply:
"When publisherId is not given" was added to test's name, but I left "(if loading = false)" to make a distinction between these and next test.